lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -205,11 +205,15 @@ int main(int argc, char* argv[]) {
   initialize_sentry();
 
   bool converged_access = false;
+  extern uint32_t session_max_rtx_count;
   // Check converged sessiond is enabled or not
   if ((config["converged_access"].IsDefined()) &&
       (config["converged_access"].as<bool>())) {
     converged_access = true;
   }
+  if (config["session_rtx_count"].IsDefined()) {
```

Comment:
What if it's not defined? Do we have a default value?

Reply:
Yes, Session manager max resending throttle count when the UPF session info version is not matching with UPF received version no.
session_rtx_count: 3

Reply:
Not sure if I understand. Do we default to a value if the config is not specified? Is 3 the default value?

Reply:
Yes, The value was added in lte/gateway/configs/sessiond.yml.

Reply:
I think it would be good to have a default value in the code as well. Or at least a warning message to indicate the value is missing.

Reply:
Ok. I will add warning message and default value as well

Reply:
Fixed it.